client/src/utils/helpers/sentry/Sentry.js

```diff
@@ -163,8 +163,8 @@ function sentryInit(
   }
 
   // Prevent sending data to Sentry when manually moving to a different URL.
-  // eslint-disable-next-line spellcheck/spell-checker, @typescript-eslint/no-unused-vars
-  window.addEventListener("beforeunload", (event) => {
+  // eslint-disable-next-line spellcheck/spell-checker
+  window.addEventListener("beforeunload", (_event) => {
```

Comment:
Remove the argument:

```suggestion
  window.addEventListener("beforeunload", () => {
```